flash/core/utilities/imports.py

```diff
@@ -74,6 +74,7 @@ def _compare_version(package: str, op, version) -> bool:
 _PYTORCHVIDEO_AVAILABLE = _module_available("pytorchvideo")
 _MATPLOTLIB_AVAILABLE = _module_available("matplotlib")
 _TRANSFORMERS_AVAILABLE = _module_available("transformers")
+_PYSTICHE_AVAILABLE = _module_available("pystiche")
```

Comment:
Should this also be added to `_IMAGE_AVAILABLE`?

Reply:
I think not doing so for now is better. I need to think about a way to segregate some deps